xml/System.Buffers/IMemoryOwner`1.xml

```diff
@@ -18,9 +18,24 @@
     </Interface>
   </Interfaces>
   <Docs>
-    <typeparam name="T">To be added.</typeparam>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <typeparam name="T">The type of elements to store in memory.</typeparam>
+    <summary>Identifies the owner of a block of memory who is responsible for disposing of the underlying memory appropriately. </summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[  
+
+The `IMemoryOwner<T>` interface is used to define the owner responsible for the lifetime management of a <xref:System.Memory%601> buffer. An instance of the `IMemoryOwner<T>` interface is returned by the <xref:System.Buffers.ArrayPool%601.Rent%2A?displayProperty=nameWithType> and <xef:System.Buffers.MemoryPool%601.Rent%2A?displayProperty=nameWithType> methods. 
```

Comment:
We don't return IMemoryOwner from ArrayPool.Rent.

Reply:
But MemoryPool.Rent does.

Reply:
also fix from xef to xref